### PR TITLE
[Feature](array) Doe fe support array

### PR DESCRIPTION
**Pull request by stalary**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:
doe support array

## Checklist(Required)

1. Does it affect the original behavior: 
    - [x] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by jackwener** [comment]
Please use `[feature](array).....` pass the ci check.

**Review by yiguolei — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsUtil.java [hunk 1]
```diff
@@ -132,16 +133,33 @@ public static boolean getBoolean(Map<String, String> properties, String name) th
...
+    public static List<String> getArrayFields(String indexMapping) {
+        JSONObject mappings = getMapping(indexMapping);
+        if (!mappings.containsKey("_meta")) {
+            return new ArrayList<>();
+        }
+        JSONObject meta = (JSONObject) mappings.get("_meta");
+        if (!meta.containsKey("doris")) {
+            return new ArrayList<>();
+        }
+        JSONObject dorisMeta = (JSONObject) meta.get("doris");
```
Comment (wuyunfeng) [inline fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsUtil.java:145]: i do not think add the name `doris`to Ealsticsearch index meta  is suitable

Comment (stalary) [reply fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsUtil.java:145]: In the automatic table creation scenario, users are required to create special mappings for the array fields. trino use it `JsonNode metaNode = nullSafeNode(mappings, "_meta");
JsonNode metaProperties = nullSafeNode(metaNode, "trino");`

**Pull request merged**